test/Directory.Build.props

```diff
@@ -2,4 +2,9 @@
   <PropertyGroup>
     <NoWarn>SA1600</NoWarn>
   </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
I know it's more maintenance, but putting these in the test projects will make it easier to maintain. Tools like `dotnet outdated` can't upgrade dependencies from external props files.

Reply:
Done. Gonna update the MS test dependencies quickly as well, then we are good for a second review run!